### PR TITLE
added hospital class and health to the miner class

### LINKED ISSUE #5: Add place, where villagers can heal wounds
Create class, which shows ?hospital? in village. All villagers can heal own wounds (restore health points).

### PR DESCRIPTION
**Pull request by CoreyT355**
closes #5

### CODE REVIEW

File: script/hospital.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    {
+      this.posX = x;
+      this.posY = y;
+      this.maxPatients = 5;
+      this.patients = [];
+    }
+
+    addPatient(patient)
+    {
+      if (this.maxPatients > 5 && patient.health > 100)
```
Comment (TomaszWychocki) [inline script/hospital.js]: Whis condition will never push new patient

File: script/hospital.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      {
+        this.patients.push(patient);
+      }
+    }
+
+    healPatient(patient) {
+      if(patient.health > 100) {
```
Comment (TomaszWychocki) [inline script/hospital.js]: Probably should be "if(patient.health < 100)"

File: script/hospital.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        patient.health++;
+      }
+      return patient;
```
Comment (TomaszWychocki) [inline script/hospital.js]: Why we need to return patient?

File: script/hospital.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+class Hospital {
```
Comment (TomaszWychocki) [inline script/hospital.js]: You must add this file to index.html

Comment (CoreyT355) [reply script/hospital.js]: Oops, sorry about the logic typo, all fixed.

Comment (TomaszWychocki) [reply script/hospital.js]: Ok, no problem :)

**Pull request merged**